packages/netlify-cms-backend-git-gateway/src/implementation.js

```diff
@@ -209,8 +209,37 @@ export default class GitGateway {
     return this.backend.getEntry(collection, slug, path);
   }
 
-  getMedia() {
-    return Promise.all([this.backend.getMedia(), this.getLargeMediaClient()]).then(
+  async loadEntryMediaFiles(files) {
```

Comment:
Large media files support was partially broken.
This adds the missing method to load relevant large media files for an unpublished entry.